packages/meditrak-app/app/assessment/selectors.jsx

```diff
@@ -3,14 +3,23 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-import { ExpressionParser, BooleanExpressionParser } from '@tupaia/expression-parser';
-import { checkAnswerPreconditionsAreMet } from './helpers';
+import {
+  ExpressionParser,
+  BooleanExpressionParser,
+} from '@tupaia/expression-parser';
+import {checkAnswerPreconditionsAreMet} from './helpers';
 
-export const getSurveyScreenIndex = state => state.assessment.currentScreenIndex;
+const expressionParser = new ExpressionParser();
```

Comment:
It takes a while to instantiate the `ExpressionParser`, and there's no need to do it multiple times as long as we clear the scope after each usage.